### PR TITLE
Removed unnecessary cors and Mvc.Razor dependency

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Issue - #2660 and #2661

### CODE REVIEW

**Comment by rynowak** [comment]
This is nice. :shipit: 

We want to additionally remove the dependency on Cors.Core in Mvc.Core. Give me a sec.

**Comment by ajaybhargavb** [comment]
Updated.

File: src/Microsoft.AspNet.Mvc.Core/project.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
         "warningsAsErrors": true
     },
     "dependencies": {
-        "Microsoft.AspNet.Cors.Core": "1.0.0-*",
         "Microsoft.AspNet.Hosting.Abstractions": "1.0.0-*",
+        "Microsoft.AspNet.Http": "1.0.0-*",
```
Comment (rynowak) [inline src/Microsoft.AspNet.Mvc.Core/project.json]: Now the question becomes, "what's this for"?

Let's merge this one and take a look at that next.

**Comment by rynowak** [comment]
:shipit: x2

**Pull request merged**